<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solitaire</class>
 <widget class="QMainWindow" name="Solitaire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Solitaire</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(78, 154, 6);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stock_deck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="waste_deck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="target_deck_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="target_deck_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="target_deck_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="target_deck_1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>75</width>
      <height>110</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset; border-width: 3px; border-color: yellow; border-radius: 5px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
